addons/l10n_it_edi/i18n/it.po

```diff
@@ -47,12 +48,8 @@ msgstr "<br/><b>Condizioni di pagamento:</b>"
 
 #. module: l10n_it_edi
 #: model_terms:ir.ui.view,arch_db:l10n_it_edi.account_invoice_form_l10n_it
-msgid ""
-"<i class=\"fa fa-warning\" role=\"img\" title=\"EDI (Italy)\"/><span "
-"class=\"mx-1\">E-invoicing (Italy)</span>"
-msgstr ""
-"<i class=\"fa fa-warning\" role=\"img\" title=\"EDI (Italy)\"/><span "
-"class=\"mx-1\">Fatturazione elettronica (Italia)</span>"
+msgid "<i class=\"fa fa-warning\" role=\"img\" title=\"EDI (Italy)\"/><span class=\"mx-1\">E-invoicing (Italy)</span>"
```

Comment:
Note that most of your diff on this file is useless and just a config difference of `polib`
Might cause conflicts in the future 🤷 
Same with updating the header of the file, the only impact is potential conflicts